core/templates/pages/donate-page/thanks-for-subscribing-modal.component.html

```diff
@@ -0,0 +1,77 @@
+<div class="container overflow-hidden rounded">
+  <div class="row">
+    <div class="col-md-6 p-0 thanks-image-container">
+      <picture>
+        <source type="image/webp"
+                [srcset]="'/assets/images/donate/thanks-for-subscribing.webp'">
+        <source type="image/jpg"
+                [srcset]="'/assets/images/donate/thanks-for-subscribing.jpg'">
+        <img alt="Education Equality Infographic"
+             class="w-100 h-100"
+             src="/assets/images/donate/thanks-for-subscribing.jpg">
+      </picture>
+    </div>
+    <div class="col-md-6">
+      <div class="row float-right mt-5 mr-2">
+        <i class="fas fa-times-circle" (click)="dismiss()"></i>
+      </div>
+      <div class="row">
+        <h3 class="thanks-subscribe-header">
+          Thanks for subscribing!
+        </h3>
+        <div class="thanks-paragraph">
+          You will start receiving updates in your inbox
+          soon. We promise not to spam, and you can
+          unsubscribe at any time.
+        </div>
+        <div class="thanks-paragraph">
+          With help and support from our community (including you!), Oppia has
+          and continues to serve the most under-resourced learners around the
+          world.
```

Comment:
Should there be translations for the strings in this file?

Reply:
Done.